src/components/shared/NavBar.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import styles from "css/shared/NavBar.module.css";
+import personLogo from "/person_logo.png";
+import fintechLogo from "/fintech_logo.png";
+import { useWindowSize } from "@uidotdev/usehooks";
+import { useClickAway } from "@uidotdev/usehooks";
+import NavBarMobile from "./NavBarMobile.tsx";
+
+function Navbar() {
+  const [showProfileOptions, setShowProfileOptions] = useState(false);
+  const size = useWindowSize(); // Get window size
+
+  const profileOptionsRef = useClickAway(() => {
+    setShowProfileOptions(false);
+  });
+
```

Comment:
@rachelsitoh  , I think the click behaviour feels kinda weird, could u change to dropdown on hover? so remove this clickaway hook and make it display on hover, should only need css, thx!